packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -81,6 +88,28 @@ export class CoreTracer implements types.Tracer {
     this.config = config;
     this.logger = this.config.logger || logger.logger();
     this.sampler = SamplerBuilder.getSampler(config.samplingRate);
+    if (config.traceParameters != null) {
+      this.numberOfAnnontationEventsPerSpan =
+          TraceParamasBuilder.getNumberOfAnnotationEventsPerSpan(
+              config.traceParameters);
+      this.numberOfAttributesPerSpan =
+          TraceParamasBuilder.getNumberOfAttributesPerSpan(
+              config.traceParameters);
+      this.numberOfMessageEventsPerSpan =
+          TraceParamasBuilder.getNumberOfMessageEventsPerSpan(
+              config.traceParameters);
+      this.numberOfLinksPerSpan =
+          TraceParamasBuilder.getNumberOfLinksPerSpan(config.traceParameters);
+    } else {
+      this.numberOfAnnontationEventsPerSpan =
```

Comment:
Could we remove this `else` block if we assign a default value to the parameters above?